discovery-provider/src/queries/get_tracks.py

```diff
@@ -151,10 +154,19 @@ def get_tracks_and_ids():
         tracks = populate_track_metadata(session, track_ids, tracks, current_user_id)
 
         if args.get("with_users", False):
-            user_id_list = get_users_ids(tracks)
-            users = get_users_by_id(session, user_id_list, current_user_id)
+            users = list(map(lambda t: t["user"][0], tracks))
```

Comment:
we have a helper that did this in a lot of other places
https://github.com/AudiusProject/audius-protocol/blob/5629dc06921a79290b4da67cd3e036ea6d199d08/discovery-provider/src/queries/get_remixable_tracks.py#L69

I think we may want to do a pass through other spots and/or consolidate that fn with this helper here.

maybe we could be smart about it and just massage the existing helper and re-use here
https://github.com/AudiusProject/audius-protocol/blob/5629dc06921a79290b4da67cd3e036ea6d199d08/discovery-provider/src/queries/query_helpers.py#L1262

Reply:
Oh nice! I didn't see that helper - wild we weren't using it in get_tracks before! Yes we should definitely reuse that one!

Reply:
I think we added that. we should just trace all the references to "with_users" it doesn't seem to be too many